service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -305,6 +311,25 @@ void shouldNotifyRepresentativesOfCMOIssued() throws Exception {
                 eq(expectedCMOIssuedNotificationParametersForRepresentative), eq("12345"));
         }
 
+        @Test
+        void shouldNotifyLocalAuthorityOfCMORejected() throws Exception {
+            CallbackRequest callbackRequest = callbackRequest();
+            CaseDetails caseDetails = callbackRequest.getCaseDetails();
+
+            given(inboxLookupService.getNotificationRecipientEmail(caseDetails, LOCAL_AUTHORITY_CODE))
+                .willReturn(LOCAL_AUTHORITY_EMAIL_ADDRESS);
+
+            given(caseManagementOrderEmailContentProvider.buildCMORejectedByJudgeNotificationParameters(caseDetails))
+                .willReturn(expectedCMORejectedNotificationParameters);
+
+            cmoNotificationHandler.notifyLocalAuthorityOfRejectedCaseManagementOrder(
+                new CaseManagementOrderRejectedEvent(callbackRequest, AUTH_TOKEN, USER_ID));
+
+            verify(notificationClient).sendEmail(
+                eq(CMO_REJECTED_BY_JUDGE_TEMPLATE), eq(LOCAL_AUTHORITY_EMAIL_ADDRESS),
+                eq(expectedCMORejectedNotificationParameters), eq("12345"));
+        }
```

Comment:
would we need a _shouldNot_ scenario?

Reply:
hmmm.. don't think so, what would we be verifying with this test?

Reply:
I have added the shouldNot scenario to the ProgressionControllerTest